examples/chat.rs

```diff
@@ -163,6 +172,8 @@ fn main() {
                 Async::Ready(None) | Async::NotReady => {
                     if !listening {
                         if let Some(a) = Swarm::listeners(&swarm).next() {
```

Comment:
We might want to consider printing all the opened ports instead of just the first one. See: https://github.com/libp2p/rust-libp2p/pull/898/files#diff-0376ea63d50ca8feb5f8bec42cb2d469R146

Reply:
Thought about that but added only the comment but might as well make that a loop.